property/index.js

```diff
@@ -166,3 +161,62 @@ module.exports = yeoman.generators.Base.extend({
 
   saveProject: actions.saveProject
 });
+
+function coerceDefault(def, value) {
```

Comment:
I think `propertyDefinition` or `propDef` should work well, I personally prefer the longer form.